### ISSUE TITLE
Exceptions are not intercepted correctly via ICommandInterceptor added via AddFilter

### ISSUE THREAD

**Issue by TricksterCat** [hadashiA/VitalRouter#105]
```
    public class ExceptionHandling : ICommandInterceptor
    {
        async ValueTask ICommandInterceptor.InvokeAsync<T>(T command, PublishContext context, PublishContinuation<T> next)
        {
            try
            {
                await next.Invoke(command, context);
            }
            catch (Exception e)
            {
                //Not enter... if add via builder.AddFilter<ExceptionHandling>();
                Debug.LogException(e);
                throw;
             }
        }
    }

    [Routes]
    //[Filter(typeof(ExceptionHandling))] - it's working!
    class SomeController 
    {
        [Route]
        private void On(SomeCommand command)
        {
            throw new Exception("Test")
        }
    }

    class AppScope : LifetimeScope
    {
        protected override void Configure(IContainerBuilder builder)
        {
          builder.RegisterInstance(Router.Default).AsImplementedInterfaces().AsSelf();
          Router.Default.AddFilter(new ExceptionHandling()); // it's not working!
        
          builder.RegisterVitalRouter(routingBuilder =>
          {
              routingBuilder.Isolated = false;
              routingBuilder.Map<SomeController>();
          });
        }
    }
```

**Comment by TricksterCat** [comment]
My way to fix this case:
(https://github.com/TricksterCat/VitalRouter/commit/5ac1d178d1ac6490f4019561cbcd0a4ba9c576f4

**Comment by hadashiA** [comment]
Thank you very much for the useful report.
I fixed it in #106!

**Comment by TricksterCat** [comment]
Thank you, everything works!
